updating thumbnailUrl and thumbnailFullUrl

avnerms opened the pull request:
we now pass a full url with an access token in it. that means it expires after 1 hour and requires additional steps to get a new url.

azurecla commented:
Hi __@happynicolle__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review this PR and note your approval/disapproval here. Thanks.

barbkess commented:
Looks like this one was overwritten with PR 4151. So, please close it to avoid confusion.

avnerms committed:
updating thumbnailUrl and thumbnailFullUrl

we now pass a full url with an access token in it. that means it expires after 1 hour and requires additional steps to get a new url.
